diagnostic.md

```diff
@@ -62,7 +62,7 @@ var result = strangeAdd(0, -5);
 
 What is the value of result?  Explain your answer.
 
-_Replace this text with your answer._
+The value is -5.  In the expression '-5' is less than '0'
 
```

Comment:
Incorrect. result equals 5. a is not greater than 0, nor is b greater than 0, therefore a-b = 5.

Always feel free to run code like this in a repl like replit or node.